### PR TITLE
Remove grpc feature from client and update README

### LINKED ISSUE #58: target attribute does not exist in `grpcClient` document
In this document: https://openliberty.io/guides/grpc-intro.html#implementing-the-unary-call

```text
Configure the grpcClient feature with the headersToPropagate and target attributes. 
```

there is no `target` attribute in the `grpcClient` document: https://openliberty.io/docs/latest/reference/config/grpcClient.html

do you refer to `host`?

Screenshot 2023-08-06 at 13 43 29

### PR DESCRIPTION
**Pull request by shin19991207**
- Addressed https://github.com/OpenLiberty/guide-grpc-intro/issues/58
  - Removed the `grpc` feature used for server-side configurations from the `query` microservice, which represents a gRPC client, for clarity and to avoids potential confusion.
  - Updated the README to mirror the code changes, correct references based on feedback, and provide clearer descriptions.

### CODE REVIEW

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]
Need to do the similar thing after the following statement for simplicity and use right endpoint. May need to confirm SME(Gordon Hutchison) exact meaning of `target`. The doc says "GRPC target endpoint". It is not clear. Is it IP or hostname or REST api endpoint?

> Configure the `grpc` element with the `target` attribute.

File: README.adoc [hunk 1]
```diff
@@ -255,7 +255,7 @@ system/server.xml
 include::finish/system/src/main/liberty/config/server.xml[]
 ----
 
-Add the [hotspot=grpc file=1]`grpc` feature to the Liberty `server.xml` configuration file. This feature enables applications running on Liberty to provide gRPC services. Configure the [hotspot=grpcConfig file=1]`grpc` feature with the `target` attribute. If you want to learn more about configuration for the `grpc` feature, see the https://openliberty.io/docs/latest/reference/config/grpc.html[GRPC Server Properties^].
+Add the [hotspot=grpc file=1]`grpc` feature to the Liberty `server.xml` configuration file. This feature enables applications running on Liberty to provide gRPC services. Configure the [hotspot=grpcConfig file=1]`grpc` element with the `target` attribute to specify the remote gRPC service's endpoint that clients should connect to for communication. For simplicity in this guide, the `target` attribute is set to a wildcard (`*`), accepting any endpoint. If you want to learn more about configuration for the `grpc` element, see the https://openliberty.io/docs/latest/reference/config/grpc.html[GRPC Server Properties^].
```
Comment (hutchig) [inline README.adoc]: I do not think it is helpful to say that ```"*"``` is 'accepting any endpoint.
Is is not the case that a ```"*"``` can be used to add additional config (such as adding an interceptor)
to ALL the targets.

File: README.adoc [hunk 2]
```diff
@@ -286,7 +286,7 @@ query/server.xml
...
+Add the [hotspot=grpcClient file=3]`grpcClient` feature to the Liberty `server.xml` configuration file for the `query` service. This feature enables gRPC client support on Liberty. Configure the [hotspot=grpcClientConfig file=3]`grpcClient` element with the `headersToPropagate` attribute to send security tokens via cookies, and the `host` attribute to specify the remote gRPC service's hostname. For simplicity in this guide, the `host` attribute is set to a wildcard (`*`), accepting any IP address. However, in a production environment, it's recommended to specify the exact hostname. If you want to learn more about `grpcClient` element configuration, see the https://openliberty.io/docs/latest/reference/config/grpcClient.html[GRPC Client Properties^].
```
Comment (hutchig) [inline README.adoc]: Personally, I think "accepting any IP address" should be something more like "applying the configuration to all remote services".

**Review by hutchig — CHANGES_REQUESTED** [review]

File: README.adoc [hunk 3]
```diff
@@ -286,7 +286,7 @@ query/server.xml
...
 ----
 
-Add the [hotspot=grpc file=3]`grpc` and [hotspot=grpcClient file=3]`grpcClient` features to the Liberty `server.xml` configuration file for the `query` service. These two features enable gRPC server and client support on Liberty respectively. Configure the [hotspot=grpcClientConfig file=3]`grpcClient` feature with the `headersToPropagate` and `target` attributes. If you want to learn more about `grpcClient` feature configuration, see the https://openliberty.io/docs/latest/reference/config/grpcClient.html[GRPC Client Properties^].
+Add the [hotspot=grpcClient file=3]`grpcClient` feature to the Liberty `server.xml` configuration file for the `query` service. This feature enables gRPC client support on Liberty. Configure the [hotspot=grpcClientConfig file=3]`grpcClient` element with the `maxInboundMessageSize` attribute to restrict inbound messages to 1024 bytes. This configuration applies universally to all gRPC client calls that get made, as indicated by the wildcard (`*`) in the `host` attribute. If you want to learn more about `grpcClient` element configuration, see the https://openliberty.io/docs/latest/reference/config/grpcClient.html[GRPC Client Properties^].
```
Comment (hutchig) [inline README.adoc]: "applies universally to all gRPC client calls that get made" ... client calls are not inbound?

**Review by hutchig — CHANGES_REQUESTED** [review]
Seems a bit confusing to me to have an Inbound config value for a clinet (outbound) call?
Perhaps better left as the headersToPropogate

**Review by hutchig — APPROVED** [review]

File: README.adoc [hunk 4]
```diff
@@ -286,7 +286,7 @@ query/server.xml
...
 ----
 
-Add the [hotspot=grpc file=3]`grpc` and [hotspot=grpcClient file=3]`grpcClient` features to the Liberty `server.xml` configuration file for the `query` service. These two features enable gRPC server and client support on Liberty respectively. Configure the [hotspot=grpcClientConfig file=3]`grpcClient` feature with the `headersToPropagate` and `target` attributes. If you want to learn more about `grpcClient` feature configuration, see the https://openliberty.io/docs/latest/reference/config/grpcClient.html[GRPC Client Properties^].
+Add the [hotspot=grpcClient file=3]`grpcClient` feature to the Liberty `server.xml` configuration file for the `query` service. This feature enables gRPC client support on Liberty. Configure the [hotspot=grpcClientConfig file=3]`grpcClient` element with the `headersToPropagate` attribute to propagate cookies. This configuration applies universally to all gRPC client calls that get made, as indicated by the wildcard (`*`) in the `host` attribute. If you want to learn more about `grpcClient` element configuration, see the https://openliberty.io/docs/latest/reference/config/grpcClient.html[GRPC Client Properties^].
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
Add the [hotspot=grpcClient file=3]`grpcClient` feature to the Liberty `server.xml` configuration file for the `query` service. This feature enables gRPC client support on Liberty. Configure the [hotspot=grpcClientConfig file=3]`grpcClient` element with the `headersToPropagate` attribute to propagate cookies. This configuration applies universally to all gRPC client calls, as indicated by the wildcard (`*`) in the `host` attribute. If you want to learn more about `grpcClient` element configuration, see the https://openliberty.io/docs/latest/reference/config/grpcClient.html[GRPC Client Properties^].
```

**Review by dmuelle — APPROVED** [review]
LGTM

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**